aviary/docs/getting_started/input_csv_phase_info.md

```diff
@@ -6,6 +6,28 @@
 This section is under development.
 ```
 
+`initial_guesses` is a dictionary that contains values used to initialize the trajectory. It contains the following keys along with default values:
+
+- actual_takeoff_mass: 0,
+- rotation_mass: .99,
+- operating_empty_mass: 0,
+- fuel_burn_per_passenger_mile: 0.1,
+- cruise_mass_final: 0,
+- flight_duration: 0,
+- time_to_climb: 0,
+- climb_range: 0,
+- reserves: 0
+
+The initial guess of `reserves` is used to define the reserve fuel. Initially, its value can be anything larger than or equal to 0. There are two Aviary variables to control the reserve fuel in the model file (`.csv`):
+- `Aircraft.Design.RESERVE_FUEL_ADDITIONAL`: the required fuel reserves: directly in lbm,
+- `Aircraft.Design.RESERVE_FUEL_FRACTION`: the required fuel reserves: given as a proportion of mission fuel.
+
+If the value of initial guess of `reserves` (also in the model file if any) is 0, the initial guess of reserve fuel comes from the above two Aviary variables.  Otherwise, it is determined by the parameter `reserves`:
+- if `reserves > 10`, we assume it is the actual fuel reserves.
```

Comment:
I'd to try and find a way to avoid this greater/less than 10 split for the initial guess too. @crecine does reserves initial guess need to exist at all anymore? Since the only way to define reserves right now don't involve "analysis" (beside a trivial equation) can we just use the user-provided variables and not make them enter that info twice?

Reply:
no, reserves is still an optional part of initial guessing